_includes/endpoint_responses/post_api_transaction.json

```diff
@@ -67,13 +46,18 @@
       "Id": "b9d0f613-985d-4a5a-8e79-a83f7b5d6b55",
       "Email": "anotheruser@example.com",
       "Mobile": "+31612345678",
```

Comment:
This should be `null`, as the request now lacks the `RequireSmsVerification` flag.